chanzuckerberg/czgenepi
# Thuang update tos pp

### Description

Update Terms and Privacy Policy copies according to https://docs.google.com/document/d/13t5cV27zA-FS5MaASvOb3DarhMn-a0C3/edit?ts=60930490#

#### Issue
https://app.clubhouse.io/genepi/story/134345/create-tos-and-privacy-pages

### Test plan

1. Visit https://thuang-tos-pp-frontend.dev.genepi.czi.technology/terms
2. Visit https://thuang-tos-pp-frontend.dev.genepi.czi.technology/privacy
3. And the copies should match the gDoc!

<img width="1363" alt="Screen Shot 2021-05-11 at 2 04 42 PM" src="https://user-images.githubusercontent.com/6309723/117884773-e2b4be80-b261-11eb-8ada-d1fa6eb1aebe.png">
<img width="1353" alt="Screen Shot 2021-05-11 at 2 04 46 PM" src="https://user-images.githubusercontent.com/6309723/117884778-e47e8200-b261-11eb-90b7-c47e2cbe63ba.png">


PTAL thank you!
